walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,13 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    return "Hello " + removeNonAlphabetChar(name.trim());
```

Comment:
I believe trim removes whitespace which is a non-alphabet character so this call is redundant.